### ISSUE TITLE
Response is looped

### ISSUE THREAD

**Issue by MEK3DK** [Azure-Samples/Cognitive-Speech-STT-Windows#36]
Started today: the OnResponseReceived event loops the same response over and over again after I have provided the first input throught the microphone. I happens when I call StartMicAndRecognition() to receive the next input.

I have not changed any code, so is anybody experiencing the same?

The only thing I did yesterday was to remote to my computer, could that cause the problem?

**Comment by hojjatabdollahi** [comment]
I have been struggling with this issue from 2 days ago. I thought it's my code's problem. Finally today I narrowed down the problem to the Speech Engine.
I've been using this for the past couple of month without error. But starting 3 days ago, it keeps repeating the same thing. I did not change anything in the code. My subscription has been the same. Everything has been the same. But suddenly it has started doing this.
